javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -319,363 +111,34 @@ public int slang_persistent() {
         public int slang_persistent_int() {
             int aggregate = 0;
             for (int i : RANDOMIZED_INDICES) {
-                aggregate ^= slangPersistent.getInt(i);
+                int[] leafUnsafe = (int[]) slangPersistent.getLeafUnsafe(i);
```

Comment:
getting the array, corresponding to the given index enables the user to avoid boxing :)
package private for now, could be used internally in `CharSeq` or `BitSet`